libs/cardano-ledger-core/src/Cardano/Ledger/Ap.hs

```diff
@@ -1,115 +0,0 @@
-{-# LANGUAGE GADTs #-}
```

Comment:
Shouldn't this removal be reflected in a version change and changelog of cardano-ledger-core, since we were exposing it as a library?

Reply:
Yeah, you are right. Thank you. I'll fix that.

Reply:
I decided to instead deprecate the module and will remove it once we have bunch of other breaking changes :slightly_smiling_face: